tests/sentry/mediators/sentry_app_installations/test_destroyer.py

```diff
@@ -139,7 +140,7 @@ def test_soft_deletes_installation(self):
     @responses.activate
     def test_deletes_on_request_exception(self):
         install = self.install
-
+        self.sentry_app.update(status=SentryAppStatus.PUBLISHED)
```

Comment:
I _think_ this change makes sense, without it it was hitting the plain return in `ignore_unpublished_app_errors` and yelling in `safe_urlread` because there was no `response.content`. This test was written in this PR: https://github.com/getsentry/sentry/pull/13984 to address not being able to uninstall a broken app, but I'm fairly certain that would only apply to a published app because if it's unpublished or internal, nobody is installing it.

Reply:
I think I'm following, but what does that mean will happen if there is broken unpublished/internal app? Can those still be installed by the devs testing/implementing them? I get that we don't raise the errors so that In our Sentry instance we don't see Internal/Unpublished errors, but how would the developers see those errors?

Reply:
I could be wrong but I don't think there is any installing happening when it's unpublished/internal, since it's only available (and is immediately available) to the organization who created it.

Reply:
I know that when you create an internal integration, it gets automatically installed, but I'm don't think that is the case for unpublished public integrations